tool/tctl/common/resource_command.go

```diff
@@ -166,8 +175,12 @@ func (u *ResourceCommand) Create(client auth.ClientI) error {
 		count++
 
 		// locate the creator function for a given resource kind:
-		creator, found := u.CreateHandlers[ResourceKind(raw.Kind)]
+		creator, found := rc.CreateHandlers[ResourceKind(raw.Kind)]
 		if !found {
+			// if we're trying to create an OIDC/SAML connector with the OSS version of tctl, return a specific error
+			if raw.Kind == "oidc" || raw.Kind == "saml" {
+				return trace.BadParameter("creating resources of type %q is only supported in Teleport Enterprise", raw.Kind)
```

Comment:
This could be a place to have a CTA and up sell for Teleport Enterprise.  Do we have any deep links to learn more for a customer?

e.g. creating resources of type %q is only supported in Teleport Enterprise. https://gravitational.com/teleport/docs/enterprise/

Reply:
If we can get a decent deeplink that could certainly be an idea. I'll leave it for now.

Reply:
I think Ben's suggestion makes sense, I would just simply add the link Ben has referenced for a start.

Reply:
Added it.